small_text/classifiers/classification.py

```diff
@@ -154,7 +160,7 @@ def __init__(self, linearsvc_kwargs=None):
         self.linearsvc_kwargs = dict() if linearsvc_kwargs is None else linearsvc_kwargs
         super().__init__(**self.linearsvc_kwargs)
 
-    def predict(self, x, return_proba=False):
+    def predict(self, x, return_proba=False, **kwargs):
```

Comment:
I don't understand this one? Is this a remnant from a previous change?

Reply:
yep, I removed it now